.npmignore

```diff
@@ -0,0 +1,14 @@
+node_modules/*
+es*/compilers/*
+.DS_Store
+.ruby-version
+_site/
+.idea/
+es5/
```

Comment:
I don't see any reason why they should be downloaded when you need just test results from npm. Source code is available from GH because npm supports GH and Git